proto/internal/temporal/server/api/persistence/v1/workflow_mutable_state.proto

```diff
@@ -38,4 +38,5 @@ message WorkflowMutableState{
     int64 next_event_id = 9;
     repeated temporal.api.history.v1.HistoryEvent buffered_events = 10;
     Checksum checksum = 11;
+    map<string, WorkflowExecutionUpdateRecord> update_records = 12;
```

Comment:
For the sake of beauty and symmetry, I would name `WorkflowExecutionUpdateRecord` as `UpdateInfo`. With all corresponding renames in entire PR. I am open to discuss this.

Reply:
Also I think it worth to add comment describing what is the key in this map.

Reply:
I made both changes.